src/test/scala/beam/integration/NetworkRelaxationSpec.scala

```diff
@@ -0,0 +1,226 @@
+package beam.integration
+
+import akka.actor.ActorSystem
+import beam.sim.config.{BeamConfig, MatSimBeamConfigBuilder}
+import beam.sim.{BeamHelper, BeamServices}
+import beam.utils.csv.{CsvWriter, GenericCsvReader}
+import beam.utils.scenario.VehicleInfo
+import beam.utils.scenario.urbansim.censusblock.entities._
+import beam.utils.{FileUtils, LoggingUtil}
+import ch.qos.logback.classic.util.ContextInitializer
+import com.google.inject.Injector
+import com.typesafe.config.ConfigFactory
+import org.matsim.core.scenario.MutableScenario
+import org.scalatest.wordspec.AnyWordSpecLike
+import org.scalatest.matchers.should.Matchers._
+
+import java.io.{Closeable, File}
+import scala.util.Random
+
+class NetworkRelaxationScenarioGenerator {
```

Comment:
I suggest moving this generator into scripts as separate class/object. Otherwise no one will ever be aware of its existence.

Reply:
done